src/shared/components/button/BigButton.tsx

```diff
@@ -1,117 +1,52 @@
-import styled from 'styled-components';
-import {
-    border,
-    BorderProps,
-    color,
-    ColorProps,
-    layout,
-    LayoutProps,
-    position,
-    PositionProps,
-    shadow,
-    ShadowProps,
-    space,
-    SpaceProps,
-} from 'styled-system';
-import { Theme } from '@/shared/theme';
-import { FlexContainer, Icon, IconDefinition, Paragraph } from '@/shared/components';
-import { flexVariant, FlexVariantProps } from '@/shared/theme/variants';
+import cn from 'classnames';
+import { Icon, IconDefinition } from '@/shared/components';
 import { isFirefox } from '@/lib/utils/isFirefox';
 
-type BaseProps = ColorProps<Theme> &
-    SpaceProps<Theme> &
-    LayoutProps<Theme> &
-    PositionProps<Theme> &
-    ShadowProps<Theme> &
-    BorderProps<Theme> &
-    FlexVariantProps;
-
-type BigButtonProps = React.ComponentPropsWithRef<typeof StyledButton> & {
+type BigButtonProps = React.ComponentPropsWithRef<'button'> & {
```

Comment:
I think this part is not relevant anymore: `React.ComponentPropsWithRef<'button'>`